### PR TITLE
Feature #789 - Login screen redesign

### LINKED ISSUE #789: Welcome screen design modifications
Goal:
To make the wallet more intuitive for IOV wallet users and to make it more easy to use.

Design:
https://www.figma.com/file/iF4KASslEFOzazuwzixKOm/Neuma-Wallet-Design-Changes?node-id=2%3A766

Changes:

- [x] Picture should be changed to given one
- [x] New IOV logo should be aligned to the middle of the picture
- [x] New text under the "Welcome to your IOV Wallet" title: To access the wallet please authenticate using Neuma or Ledger Nano S
- [x] Two "Continue..."buttons are put in the middle of the page
- [x]  First button: Continue with Neuma Browser Extension
- [x]  Second button: Continue with Ledger Nano S
- [x] after the buttons: "or, in case you don’t have the extension"
- [x] "Get Neuma Browser Extension" and arrow that links to neuma.io in a new tab

Question:
- [x] The two buttons have a different color schema than used for governance app, etc. Should these colors be used for the Wallet only? Or should we use the color schema every other app uses?
- [x] Andrei - provide set of icons for devs

Wallet - Welcome pa (1)

### PR DESCRIPTION
**Pull request by albertandrejev**
**Description**
This PR will implement new design of the Login screen. Closes #789.

**New look**
login

### CODE REVIEW

File: packages/sanes-browser-extension/package.json [hunk 1]
```diff
@@ -49,9 +49,9 @@
     "@iov/keycontrol": "^1.0.0-alpha.1",
     "@iov/lisk": "^1.0.0-alpha.1",
     "@iov/multichain": "^1.0.0-alpha.1",
-    "@material-ui/core": "^4.3.1",
-    "@material-ui/icons": "^4.2.1",
-    "@material-ui/styles": "^4.3.0",
+    "@material-ui/core": "^4.5.0",
+    "@material-ui/icons": "^4.4.3",
+    "@material-ui/styles": "^4.5.0",
```
Comment (webmaster128) [inline packages/sanes-browser-extension/package.json]: Can you rebase this onto master where those library upgrades should already be included?

Comment (albertandrejev) [reply packages/sanes-browser-extension/package.json]: Unfortunately I forgot to include `sanes-browser-extension` `package.json` to PR with packages upgrade.

File: packages/bierzo-wallet/src/routes/login/components/index.tsx [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+          >
+            Get Neuma Browser Extension
+          </Button>
+        </Block>
+      </Block>
+    </Block>
+  );
+};
+
+export default PageColumn;
```
[the remark this answers is not in the document]
Comment (albertandrejev) [reply packages/bierzo-wallet/src/routes/login/components/index.tsx]: I'd better use different name for the component, like in other routes - `Layout` to keep consistency.

[the remark this answers is not in the document]
Comment (webmaster128) [reply packages/bierzo-wallet/src/routes/login/components/index.tsx]: hmm, but when you search for `Layout`, you don't find what you search for. Also the component name should be in the file name somehow.

[the remark this answers is not in the document]
Comment (albertandrejev) [reply packages/bierzo-wallet/src/routes/login/components/index.tsx]: You are right!

**Pull request merged**